lifecycle-operator/controllers/options/keptnconfig_controller.go

```diff
@@ -176,29 +184,27 @@ func (r *KeptnConfigReconciler) reconcileKeptnGateway(ctx context.Context, confi
 		err := r.Client.Create(ctx, getKeptnGatewayDeployment(r.Namespace))
```

Comment:
In this case the namespace where the keptn-gateway will be deployed is the same as the namespace, where KeptnConfig is applied, is this really what we want? I guess we always want to deploy it in the same namespace as keptn is deployed, no?

Reply:
That is the namespace of the reconciler, which should be the namespace where keptn is deployed :)

Reply:
no that is the namespace of the request of the keptnconfig which was applied to the cluster, so if you apply keptnconfig to the "default" namespace, the deployment will be deployed to the "default" namespace, which is not correct